openedx/features/partners/models.py

```diff
@@ -26,6 +26,22 @@ class Partner(TimeStampedModel):
     slug = models.CharField(max_length=100, unique=True, help_text="A unique identifier for an organization")
     email = models.EmailField(help_text="Contact email of an organization")
     configuration = JSONField(null=False, blank=True, default=dumps({"USER_LIMIT": ""}))
+    main_title = models.CharField(blank=True, max_length=100, help_text="Display on partners landing page as main title.")
+    main_description = models.TextField(blank=True, help_text='Brief description of partners landing page.')
```

Comment:
please be consistent with the use of ' or "

Reply:
fix